docs/source/developers/guide/introduction.rst

```diff
@@ -0,0 +1,142 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+
+.. SCOPE OF THIS SECTION
+.. Make an introduction to the Guide for New Contributors.
+.. Add an introduction to the project, why to get involved
+.. and a quick checklist of the contributing process.
+.. Add the Table of Contents. All detailed explanations
+.. should be added as a link in the content.
+
+
+.. _introduction:
+
+***********************
+New Contributor's Guide
+***********************
+
+This guide is meant to be a resource for contributing to
+`Arrow <https://arrow.apache.org//>`_ for new contributors.
+
+Starting to contribute to a project like Apache Arrow can look
+scary. Taking small steps will help making this tack easier.
```

Comment:
I would rather say "intimidating" rather than "scary". The negative wording can be offputting or worrying.

(btw, I don't know what a "tack" is in this context? the explanations I find seem unrelated to software)

Reply:
It wants to be a motivational intro =)